tests/test_project.py

```diff
@@ -212,6 +212,14 @@ def test_vcs_git_repo(mocker, package_no_version):
     assert str(trampolim._build.Project().version) == '1.0.0.23+gea1f213'
 
 
+def test_vcs_git_repo_exact(mocker, package_no_version):
+    mocker.patch(
+        'subprocess.check_output',
+        side_effect=[b'v2.1.1-0-g5ce8958'],
```

Comment:
The whole point of the patch is so that the output will always look something like this; adding `--dirty` will always produce the last two items. The tests originally didn't have the last two items.

Reply:
Now it might be broken. :) I can look into it in a little bit.